src/main/java/org/proshin/finapi/primitives/pair/SpaceSeparatedPair.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2018 Roman Proshin
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.proshin.finapi.primitives.pair;
+
+public final class SpaceSeparatedPair extends UrlEncodedPair {
+    public SpaceSeparatedPair(String name, String value) {
+        super(name, value);
+    }
+
+    @Override
+    public String getValue() {
+        return super.getValue().replaceAll("\\+", " ");

```

Comment:
it seems that `+` is the right encoding for space symbols: https://stackoverflow.com/questions/1634271/url-encoding-the-space-character-or-20
I think the real issue is that the "plus" symbol is encoded one more time for some reason. I've checked the code quickly but couldn't identify the problem. Could you check it please?